README.md

```diff
@@ -157,6 +157,7 @@ The configuration includes the following parameters:
   * `maxScale` (Optional): maximum number of instances this function can scale to (default: 20)
   * `maxConcurrency` (Containers only, Optional): Concurrency defines the number of simultaneous requests your container can handle at the same time (default: 50)
   * `memoryLimit`: RAM allocated to the function instances. See the introduction for the list of supported values
+  * `cpuLimit`: (Containers only) CPU allocated to the container instances. Only values ranging from 70mvCPU to 2240mvCPU (included) are accepted.
```

Comment:
in order to have a single source of truth for those values, can you ensure docs-content contains the range of theses values and add the link here ?

Reply:
Done